recipes/spade/conda_build_config.yaml

```
@@ -0,0 +1,10 @@
+hdf5:
+  - 1.12.2
+  - 1.14.3
+numpy:
+  - 1.26
+  - 2.0  #Lowest compatible version
```

I think something in the numpy spec conflicts with the python variants? I didn't study the log completely. I also don't see any direct numpy imports in spade. Is numpy a requirement at all?

If we keep numpy as a requirement, maybe drop to the conda-forge recommended spec?

```suggestion
numpy:
  - 2.0
```